cmd/mutagen/main.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"github.com/mutagen-io/mutagen/pkg/mutagen"
```

Comment:
Can you relocate this import down below, just above `github.com/mutagen-io/mutagen/pkg/prompting` (but maintain the blank line between the `github.com/mutagen-io/mutagen/cmd/...` and `github.com/mutagen-io/mutagen/pkg/...` imports)?